File: week-04/day-2/src/garden/Flower.java
package garden;

public class Flower extends Plant {

    int numberOFFlowers;

    public Flower(int numberOFFlowers, String color, int waterNeed, double waterAbsorbtion){
        super(color , waterNeed, waterAbsorbtion);
        this.numberOFFlowers = numberOFFlowers;
        this.waterAbsorbtion = 0.75;
    }

    public  void status(){

    }
}

File: week-10/day-4/Garden/Garden/Flower.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Garden 
{
    class Flower : Plant {

        public Flower(String color, double waterNeed, double waterAbsorbtion) : base(color, waterNeed, waterAbsorbtion) { 
        }

        public override void Status(){
            if (waterNeed < 5){
                Console.WriteLine("The " + color + " " + this.GetType().Name + " needs water.");
            }
            else {
                Console.WriteLine("The " + color + " " + this.GetType().Name + " doesn't needs water.");
            }

            
        }
    }
}
